TelegramBot.py

```diff
@@ -56,13 +57,13 @@ def ping_server(self):
 
 
     def goto_state_running(self):
-        logging.debug("Logging state is turned on running again for " + self.name)
+        logging.debug("{} Logging state is turned on running again for {}".format(self.owner, self.name))
```

Comment:
I don't think that the telegram admin name is particularly useful in the debug logs, is it? I propose do remove it.